app/components/ui/forms.tsx

```diff
@@ -84,41 +76,3 @@ export function TextArea({ label, ...props }: TextAreaProps) {
     </label>
   );
 }
-
-type ButtonProps = ButtonHTMLAttributes<HTMLButtonElement>;
-
-export function Button({ children, ...props }: ButtonProps) {
-  return (
-    <button {...props} className="border-2 bg-red-700 text-white rounded-lg p-2" type="submit">
-      {children}
-    </button>
-  );
-}
```

Comment:
remove the button too. 
to reduce duplications.
make sure no other components using it.